eth_pydantic_types/hex.py

```diff
@@ -28,6 +36,7 @@
 
 
 class BaseHex:
```

Comment:
So I am working on core right, and I recently realized I really need a `HexInt` class.
Basically it allows any hex-str or or bytes but always validates into an integer.

I am going to make one in core ape for now but eventually we can use this work to double-import.